src/web/src/components/LanguageSelector.tsx

```diff
@@ -0,0 +1,53 @@
+import { createContext, ChangeEvent, useState } from 'react';
+import {
+  createStyles,
+  FormControl,
+  Grid,
+  InputLabel,
+  ListItemSecondaryAction,
+  MenuItem,
+  Select,
+} from '@material-ui/core';
+
+import changeLang from '../translations/i18n';
+
+export interface TranslationInterface {
+  currentLanguage: string;
+}
+
+export const translationInterface = createContext<TranslationInterface>({
+  currentLanguage: 'en',
+});
+
+const languages: Array<{ name: string; code: string }> = [
+  { name: 'English', code: 'en' },
+  { name: 'Español', code: 'es' },
+];
+
+const LanguageSelector = () => {
+  // Default language to translate is en
+  const [language, setLanguage] = useState('');
+
+  const changeLanguage = (
+    e: ChangeEvent<{ name?: string | undefined; value: string | unknown }>
+  ) => {
+    e.preventDefault();
+    setLanguage(e.target.value as string);
+    changeLang(e.target.value);
+  };
+
+  return (
+    <Grid container>
+      <FormControl>
+        <InputLabel>Language</InputLabel>
+        <Select value={language} label="Language" onChange={changeLanguage}>
+          {languages.map((index) => {
+            return <MenuItem value={index.code}>{index.name}</MenuItem>;
```

Comment:
🚔 React crime 🚔 , each child of a list  should have a unique [key](https://reactjs.org/docs/lists-and-keys.html).
```suggestion
            return <MenuItem value={index.code}>{index.name}</MenuItem>;
```

Reply:
I think @DukeManh talk about the `key` props